src/chat/src/Bridge/Symfony/SessionStore.php

```diff
@@ -9,10 +9,10 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\AI\Agent\Chat\MessageStore;
+namespace Symfony\AI\Chat\Bridge\Symfony;
```

Comment:
maybe rather `HttpFoundation`?

```suggestion
namespace Symfony\AI\Chat\Bridge\HttpFoundation;
```